File: code/5e102.pas
Program Bai102;
var n, dem, c: integer;
begin
  readln(n);
  dem := 0;
  while n > 0 do 
    begin
    c := n mod 10;
    if c = 0 then dem := dem + 1;
    n := n div 10;
    end;
  write(dem);
end.

File: code/5e102.cpp
#include <iostream>

using namespace std;


int main() 
{

    int n,dem,c;

    cin>>n;

    dem=0;

    while (n>0){

        c=n%10;

        if (c==0)

            dem++;

        n=n/10;

    }
    
    cout<<dem;
return 0;

}